docker/Dockerfile-ubuntu-20

```diff
@@ -10,7 +10,14 @@ RUN apt update && \
         ninja-build \
         python3-pip \
         python3-venv \
-        python3-dev curl gnupg apt-transport-https && \
+        python3-dev curl gnupg apt-transport-https \
+        zlib1g \
+        wget && \
+    wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb && \
```

Comment:
May be we can wrap all cs dependeces intro one script

Reply:
https://hub.docker.com/_/microsoft-dotnet-sdk

Reply:
> https://hub.docker.com/_/microsoft-dotnet-sdk

But it is not possible to run docker commands inside a dockerfile. Or is it?

Reply:
Just use it for `FROM` directive at the top and start with the container, full of M$ libs.

Reply:
In this case it should be a separate dockerfile isn't it?
cc @CLTanuki @OrangeSmokingJacket